node/src/RedisClient.ts

```diff
@@ -127,6 +128,20 @@ export class RedisClient extends BaseClient {
      * If not provided, the server will respond with "PONG".
      * If provided, the server will respond with a copy of the message.
      * @returns - "PONG" if `message` is not provided, otherwise return a copy of `message`.
+     *
+     * @example
+     * ```typescript
+     * // Example usage of ping method without any message
+     * const result = await client.ping();
+     * console.log(result); // Output: "PONG"
+     * ```
+     *
+     * @example
+     * ```typescript
+     * // Example usage of ping method with a message
+     * const result = await client.ping("Hello");
+     * console.log(result); // Output: "Hello"
```

Comment:
minor: please be consistent with one or two quotes, here it's 
 > // Output: "Hello"

and other places it's:
>  // Output: 'Connection Name'